problem_builder/public/js/step.js

```diff
@@ -4,6 +4,8 @@ function MentoringStepBlock(runtime, element) {
 
     var submitXHR, resultsXHR,
         message = $(element).find('.sb-step-message');
+    
+    var childManager = new ProblemBuilderUtilLMS.ChildManager(element, runtime);
```

Comment:
Done. I went for ``ProblemBuilderStepUtil`` this is a global name attached to window object, so I'd rather be verbose than have a clash.